packages/bitv/bitv.1.2/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1"
+name: "bitv"
+version: "1.2"
+maintainer: "filliatr@lri.fr"
+authors: ["Jean-Christophe Filliâtre"]
+homepage: "https://www.lri.fr/~filliatr/software.en.html"
+license: "LGPL v2"
+build: [
+  ["autoconf"]
+  ["./configure"]
+  [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "bitv"]
+depends: [
+  "ocamlfind"
+  "conf-autoconf" {build}
+]
```

Comment:
Needs `available: [ocaml-version >= "4.00.1"]` due to use of `iteri`.

On OCaml 3.12.1: 
```
# File "bitv_string.ml", line 127, characters 2-13:
# Error: Unbound value Bytes.iteri
# make: *** [bitv_string.cmo] Error 2
```